### ISSUE TITLE
Watching studios

### ISSUE THREAD

**Issue by mybearworld** [ScratchAddons/ScratchAddons#6375]
### Why this would be helpful

Studios are really the only thing you can create that you can't get comment notifications from. It would be great if you could do this for studios as well.

### How this addon works

This addon would add a button to studios that would allow you to watch the studio. On every new comment, it would add a message to the messages page or the pop up.

### Additional context

I'm probably forgetting something important, here.

**Comment by rgantzos** [comment]
A few things to think about:
1. If you choose to watch a lot of studios, then it could spam the API a bit
2. Spreading out API calls could lead to missing notifications if a comment was deleted, and could also lead to a lot of notifications coming at once
3. On project comments/profile comments, you get notifications for replies even if the comment isn't replying to you- if this was added to studios, then it would require fetching all comments/replies and storing a lot of information (storing which replies have been read, and which ones haven't)

**Comment by Secret-chest** [comment]
Just use followed studios!

**Comment by mybearworld** [comment]
Those give you studio activity, though.

**Comment by Secret-chest** [comment]
> Those give you studio activity, though.

No, following studios does nothing, you get activity for studios you curate.

**Comment by mxmou** [comment]
> Just use followed studios!

I don't think reusing an existing Scratch feature for a different purpose is a good idea. Following studios already does something: it adds the studio to the "Studios I'm Following" row on your profile. It's also meant to display projects from that studio on the front page, but that has been broken for a long time.

**Comment by WorldLanguages** [comment]
Related: #6082, #3115

**Comment by WorldLanguages** [comment]
> This addon would add a button to studios that would allow you to watch the studio. On every new comment, it would add a message to the messages page or the pop up.

Maybe we could reuse the "studio activity" message and create an addon that "sends you a studio activity message when there's new studio activity" instead of just once a day. It wouldn't actually send you messages, as they would be fake and handled by this addon, but it would be a nice way to make it somewhat intuitive.

**Comment by WorldLanguages** [comment]
> * If you choose to watch a lot of studios, then it could spam the API a bit

Yeah, if you watch too many studios, each one would be checked less often, so we should add a limit
